data/share/tweaks/amd.json

```diff
@@ -0,0 +1,9 @@
+{
+	"id": "amd_radeon_aco_vulkan",
+	"name": "AMD: Radeon ACO Vulkan Compiler",
+	"description": "Enable ACO based Vulkan shader compiler for RADV instead of LLVM",
+	"url": "https://github.com/mesa3d/mesa",
```

Comment:
Maybe this link should explain what this option does instead of linking to mesa source.

Reply:
@tkashkin - I've mainly been following Phoronix with artices like this: https://www.phoronix.com/scan.php?page=article&item=radv-aco-feb2020